ebean-postgis/src/main/java/io/ebean/postgis/ScalarTypePgisLineString.java

```diff
@@ -3,10 +3,20 @@
 import io.ebean.config.dbplatform.ExtraDbTypes;
 import org.postgis.LineString;
 
+import java.sql.SQLException;
+
 public class ScalarTypePgisLineString extends ScalarTypePgisBase<LineString> {
 
   public ScalarTypePgisLineString() {
     super(ExtraDbTypes.LINESTRING, LineString.class);
   }
 
+  @Override
+  public LineString parse(String value) {
+    try {
+      return new LineString(value);
+    } catch (SQLException e) {
+      throw new IllegalStateException(e);
```

Comment:
I was wondering if it would make sense to include the erroneous value here as message argument, but as far as I can see LineString is already throwing a SqlException with the value included in the message

Reply:
As I think, this error should never occur. String `value` passed to constructor is generated with this same code (`LineString#toString()`). This could happen, when WTK string would be from external source, but this is not the case.